### DISCUSSION TITLE
Speech to Text language

### DISCUSSION THREAD

**Discussion by hmjk99** [deepgram/community#945] [General help] [answered]
I am using deepgram/s speech to text js sdk in our chat widget application. I want the language to be from the user's browser, but I noticed deepgram does not support all languages. So I was wondering if there is a way to know if a certain language is supported through the sdk or api so i could set 'en' as the default fallback language.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @hmjk99, we do have a [GET /models](https://developers.deepgram.com/docs/model-metadata#get-models) endpoint, that you could make work for this purpose. It fetches metadata on all publicly available models, including their language support. Alternatively, you could put logic in place based on our documented [models and languages](https://developers.deepgram.com/docs/models-languages-overview), but that may change and grow over time, so you'd have to keep it up-to-date periodically as needed.
